src/crawlee/storages/_request_source.py

```diff
@@ -0,0 +1,84 @@
+from __future__ import annotations
```

Comment:
You are talking about the request queue in the docstrings (I'm talking about the whole file)